packages/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -33,71 +26,70 @@ const mergeProps = makeMergeProps<PopoverState>({});
  * @param defaultProps - (optional) default prop values provided by the implementing type
  */
 export const usePopover = (props: PopoverProps, defaultProps?: PopoverProps): PopoverState => {
-  const state = mergeProps(
-    {
-      size: 'medium',
-      open: (undefined as unknown) as boolean, // mergeProps typings require this
-      setOpen: () => null,
-      triggerRef: { current: null },
-      contentRef: { current: null },
-      arrowRef: { current: null },
-      children: null,
-      setContextTarget: () => null,
-      contextTarget: undefined,
-    },
-    defaultProps,
-    props,
-  );
+  const [contextTarget, setContextTarget] = usePopperMouseTarget();
+  const initialState = {
+    size: 'medium',
+    contextTarget,
+    setContextTarget,
+    ...props,
```

Comment:
`props` should not be spreaded into the `state` object

Reply:
In this case, they should since `Popover` itself does not render DOM and there are no concept of slots